sqldelight-gradle-plugin/build.gradle

```diff
@@ -51,6 +52,7 @@ dependencies {
     targetConfiguration = "shadow"
   }
 
+  implementation libs.sqlPsiEnvironment
```

Comment:
This adds all IntelliJ classes to the gradle plugin classpath, which can cause many problems. We should use worker classpath instead

Reply:
Ignore it, IntelliJ is compileonly, Not implementation/api 🤦🏻‍♂️